content/sensu-go/5.2/guides/install-check-executables-with-assets.md

```diff
@@ -10,153 +12,119 @@ menu:
     parent: guides
 ---
 
-- [How to create a check that depends on an asset](#how-to-create-a-check-that-depends-on-an-asset)
-- [How to create a handler that depends on an asset](#how-to-create-a-handler-that-depends-on-an-asset)
-- [How to create a mutator that depends on an asset](#how-to-create-a-mutator-that-depends-on-an-asset)
+- [1. Download an asset definition from Bonsai](#1-download-an-asset-definition-from-bonsai)
```

Comment:
Noticed that these items have numbers to them. Is that intentional? I just haven't see us use it anywhere else.

Reply:
Yep, it's intentional. In this case, it's a very 1-2-3 kind of process.